addons/mail/models/mail_thread.py

```diff
@@ -1684,9 +1684,15 @@ def _message_post_process_attachments(self, attachments, attachment_ids, message
             # taking advantage of cache looks better in this case, to check
             filtered_attachment_ids = self.env['ir.attachment'].sudo().browse(attachment_ids).filtered(
                 lambda a: a.res_model == 'mail.compose.message' and a.create_uid.id == self._uid)
+            # update filtered (pending) attachments to link them to the proper record
+
```

Comment:
Strange new line here. I would not use any, because the comment is for explaining the `if` that is below.
(And if I do want to use a new line, then I would put it above the comment, not below; and I would add one also above the other comment for consistency.)